content/en/status.md

```diff
@@ -108,9 +108,10 @@ same as the **Protocol** status.
 - **API:** draft
 - **SDK:** draft
 - {{% spec_status "Protocol" "protocol/otlp" "Logs" %}}
+  - {{% spec_status "Data model" "logs/data-model" %}}
```

Comment:
Two notes here:

- By using `spec_status` the data-model status will get automatically extracted from the data-model spec page.
- I've put the **Data model** status entry as a subentry of **Protocol** since, as the note below explains, it "is released as part of the OTLP protocol". Does that work for you @tigrannajaryan? If not, I can put it at the top level. I'm open to other suggestions too.

Reply:
We don't have this for other signals, although metrics has a data model page. Perhaps remove for logs as well?

Reply:
It's your call. I can remove it. This was just a suggestion.

Reply:
The text is fine either way. My concern is that it is inconsistent: we talk about logs data model, but not about data models of traces and metrics. We should either do it for all 3 signals or delete this one.
I have a slight preference to delete, since I don't think it adds much value.

Reply:
+1 to delete then